pkg/workload/histogram/exporter/openmetrics_exporter.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package exporter
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/codahale/hdrhistogram"
+	"github.com/prometheus/common/expfmt"
+)
+
+// OpenmetricsExporter exports metrics in openmetrics format
+type OpenmetricsExporter struct {
+	writer *bufio.Writer
+	Labels *map[string]string
+}
+
+func (o *OpenmetricsExporter) Validate(filePath string) error {
+	if strings.HasSuffix(filePath, ".json") {
```

Comment:
Both openmetrics exporter and json exporter have the same validation function?
If yes, then probably its invariant of exporter type and can be moved to utils.

Reply:
Sorry I didn't understand this comment.

Reply:
Implementation of Validate() method is same for both receiver types.
They are both validating that suffix is json. Could this Validate() method be moved out of the interface.

Reply:
I made this design more of thinking in the future in case we need to do more validations. As of noe, checking for json file seemed appropriate to me.